pages/design/tools/figma.mdx

```diff
@@ -1,6 +1,110 @@
+
+<Text textAlign="center">
+   <Flex alignItems='center' flexDirection='row' mt='4' mb='4'>
+   <Box width={6/12} mr='4'>
+     <BorderBox height="250px" overflow='hidden'>
+      <img src="https://user-images.githubusercontent.com/6846673/61830583-5af98800-ae20-11e9-8aa9-459fc2d51133.gif"/>
+     </BorderBox>
+     <Text textAlign='center' color='gray.6'>Search for components in the assets menu and drag them onto the canvas.</Text>
+   </Box>
+    <Box width={6/12} mr='4'>
+      <BorderBox height="250px" overflow='hidden'>
+       <img src="https://user-images.githubusercontent.com/6846673/61830581-5af98800-ae20-11e9-93ab-9c33ff0dd2be.gif"/>
+      </BorderBox>
+      <Text textAlign='center' color='gray.6'>Some components have multiple states, which you can toggle in the layer list.</Text>
+    </Box>
+   </Flex>
+</Text>
 
-## Components
+
+## <a name="Prototyping"></a>Prototyping and Wireframing
+We’ve developed a prototyping toolkit that includes a set of page templates and components to help you get started with exploration and prototyping in Figma. These pages are “medium” fidelity and have flexible elements that can be hidden or shown to work in different complexities. You can use the “Prototype” tab in the righthand sidebar to get started with interactive prototype projects. When you’re ready to present, simply hit the play (▶️) button in the top right.
+
+
+<Text textAlign="center" fontStyle='italic'>
+   <Flex alignItems='center' flexDirection='row' mt='4' mb='4'>
+    <Box width={12/12}>
+      <BorderBox overflow='hidden'>
+       <img src="https://user-images.githubusercontent.com/6846673/61732575-bfd6b480-ad32-11e9-8e77-5a8c28b9bd88.png"/>
+      </BorderBox>
+    </Box>
+   </Flex>
+</Text>
+
+Duplicate this [Figma file](https://www.figma.com/file/ex9hnSDJTr8fG24cTMyRJzHD/prototyping-pages?node-id=0%3A1) to your own project to use these templates for prototyping or exploration.
+
+## <a name="Collaboration"></a>Collaboration
+All Hubbers have read-only access to all GitHub design files and edit access to files within their team. You can leave [Figma comments](https://help.figma.com/article/44-comments) to provide feedback within Figma and check [issues](https://help.github.com/en/articles/about-issues) to track decisions and progress. The design team works with incremental correctness, so share early and share often!
+
+## <a name="Resources"></a>Resources
+Check out these places to learn more about Figma and GitHub:
+- [Figma Design Documentation](https://help.figma.com/)
+- [GitHub takes its collaborative culture to a new level](https://www.figma.com/blog/github-case-study/)
+- [GitHub Primer Design](https://primer.style/design)
+
+List of current Primer Components:
```

Comment:
We talked irl about this, but I think the list of components could come first here